Alicerce/Sources/View/UIColor.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  UIColor.swift
+//  Alicerce
+//
+//  Created by Luís Portela on 17/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import UIKit
+
+public extension UIColor {
+    private static let divisor = CGFloat(255)
+
+    private typealias Components = (red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat)
+
+    public convenience init(hex: String) {
+        let hex = hex.trimmingCharacters(in: .whitespacesAndNewlines)
+            .replacingOccurrences(of: "#", with: "")
+
+        var hexValue: UInt32 = 0
+
+        guard Scanner(string: hex).scanHexInt32(&hexValue) else {
+            fatalError("😱 Cannot convert string into `UInt32`")
```

Comment:
I also don't particularly like of throwing on these types of initialisers, because it requires more boilerplate when using, but I asked this question since API wise it would perhaps be the **right thing ®** 😄 

However, Apple also traps on invalid input (e.g. duplicate keys on dictionaries), so we might be safe to simply crash here,. which makes using the API much easier 🤔